### DISCUSSION TITLE
Relations

### DISCUSSION THREAD

**Discussion by dwrinn** [aidenlx/obsidian-zotero#324] [Help]
Is there any way to pull in the relationship to other documents in zotlit?

**Comment by aidenlx** [comment]
Yes, in v2. `zt.relatedItems` exposes everything from Zotero's Related panel, and it is wired up out of the box: the default frontmatter includes a `related` field set to `zt.relatedItems | note_links`, which writes wiki-links to those items' literature notes.

Related items that do not have a note yet render a visible marker rather than being dropped silently, so you can tell what is missing.

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
